packages/vue-instantsearch/src/components/Pagination.vue

```diff
@@ -1,5 +1,8 @@
 <template>
-  <div v-if="state" :class="suit()">
+  <div
+    v-if="state"
+    :class="{ [suit()]: true, [suit('', 'noRefinement')]: state.nbPages <= 1 }"
```

Comment:
This is additive, so it shouldn't be considered breaking IMO.

Worst case, if users have snapshot tests they can break, but this won't break production.